cmd.go

```diff
@@ -240,6 +270,23 @@ func (b *Bot) executePassiveCommands(cmd *PassiveCmd) {
 	wg.Wait()
 }
 
+func (b *Bot) executeFilterCommands(cmd *FilterCmd) (string, error) {
+	var err error
+	newMessage := cmd.Message
+	for k, filter := range filterCommands {
+		switch filter.Version {
+		case fv1:
+			newMessage, err = filter.FilterFuncV1(cmd)
+			if err != nil {
+				b.errored(fmt.Sprintf("Error executing filter %s", k), err)
+				continue
+			}
+			cmd.Message = newMessage
+		}
+	}
+	return cmd.Message, nil
+}
```

Comment:
See the function that calls this method, I don't think it's necessary for it to return an error, it could default do return the original message and log the error, something linke this:

```
func (b *Bot) executeFilterCommands(cmd *FilterCmd) string {
	for k, filter := range filterCommands {
		switch filter.Version {
		case fv1:
			filtered, err := filter.FilterFuncV1(cmd)
			if err != nil {
				b.errored(fmt.Sprintf("Error executing filter %s", k), err)
				continue
			}
			cmd.Message = filtered
		}
	}
	return cmd.Message, nil
}
```